iris/service/content/file/document.py

```diff
@@ -18,3 +25,43 @@ class File(Document):
         default=dc_defaults(DC_CREATED, DC_MODIFIED),
         doc="Dublin Core data."
     )
+
+    state = Property(
+        doc="The state of the file (visible/hidden)"
+    )
+
+    original_name = Property(
+        doc="The original file name before upload"
+    )
+
+    owner_id = Property(
```

Comment:
This is a relation to an owner and should be declared as such.
That will then automatically allow to provide the full owner via the REST API.